docs/sql-reference/functions-reference/slice.md

```diff
@@ -1,19 +1,19 @@
 ---
 layout: default
-title: SLICE
-description: Reference material for SLICE function
+title: ARRAY-SLICE

```

Comment:
@yuval-boker it is ARRAY_SLICE not ARRAY-SLICE correct? you have put both in here

Reply:
Right, sorry. Should be `ARRAY_SLICE`